Jamulus.pro

```diff
@@ -301,6 +301,16 @@ DISTFILES_OBOE += libs/oboe/AUTHORS \
         android/AndroidManifest.xml \
         android/sound.h \
         android/sound.cpp
+
+    isEmpty(PREFIX) {
+        PREFIX = /usr/local
+    }
+    isEmpty(BINDIR) {
+        BINDIR = bin
+    }
+    BINDIR = $$absolute_path($$BINDIR, $$PREFIX)
+    INSTALLS += target
+    target.path = $$BINDIR
```

Comment:
This will likely need to be swapped:

```suggestion
    target.path = $$BINDIR
    INSTALLS += target
```

Reply:
Not sure if this is valid syntax but it might also be nicer to use more descriptive target name:

```suggestion
    jamulus_bin_target.path = $$BINDIR
    INSTALLS += jamulus_bin_target
```

Reply:
Build & install worked when I checked, but I'd be surprised if swapping them broke it

Reply:
Reading https://doc.qt.io/qt-5/qmake-advanced-usage.html#installing-files, `target` seems to be a magic name that will choose something in unspecified manner. For the other installed files, you will need to specify `.files`.